### ISSUE TITLE
Hello, may I ask if I can use the pre trained model you provided for my own test set

### ISSUE THREAD

**Issue by wuzuyin** [zhengyuf/IMavatar#51]

**Comment by zhengyuf** [comment]
You can animate pre-trained models with novel expressions from your own test video. But you have to train the model to obtain a new avatar.
